src/main/java/org/openrewrite/java/migrate/util/ListFirstAndLast.java

```diff
@@ -34,6 +34,7 @@
 
 public class ListFirstAndLast extends Recipe {
 
+    // `List` has `get`, `add`, and `remove` methods that take an index, even if more SequencedCollections have `*First` and `*Last` methods
```

Comment:
I don't understand this comment, maybe it can be clarified? (I'm not up to date on Sequenced collections though)

Reply:
Sure, thanks for the feedback; clarified it a bit in [6d8bbb8](https://github.com/openrewrite/rewrite-migrate-java/pull/424/commits/6d8bbb8ad72429d24c63f227caea21bb57e0d1d1)